docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -378,17 +378,16 @@ to the required ECR repositories.
 
 :::caution
 This method of authentication is a "lowest common denominator" approach that
-will work regardless of where Kargo is deployed. i.e. If running Kargo outside
-of EKS, this method will still work.
+will work regardless of where Kargo is deployed. i.e. if running Kargo outside EKS, this method will still work.
 
-If running Kargo within EKS, you may wish to consider using EKS Pod Identity
+If running Kargo within EKS, you may wish to either consider using EKS Pod Identity or IRSA
 instead.
 :::
 
-#### EKS Pod Identity
+##### EKS Pod Identity (default)
```

Comment:
In an attempt to say more with fewer words, I would consider re-titling this section as "EKS Pod Identity or IAM Roles for Service Accounts (IRSA)" and then make _small_ tweaks to the text of this section. The approach to both is so very similar, that it shouldn't be too hard to cover these two as a pair.

IRSA requires some additional configuration when the controller is installed and those details can be called out in an `:::info` or `:::note` box.